jbmc/src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -2213,7 +2208,7 @@ void java_bytecode_convert_methodt::convert_invoke(
     symbol.pretty_name = id2string(arg0.get(ID_C_class)).substr(6) + "." +
                          id2string(symbol.base_name) + "()";
     symbol.type = arg0.type();
-    symbol.type.set(ID_access, ID_public);
+    symbol.type.set(ID_access, ID_private);
```

Comment:
It's not guaranteed. It can well be private. The previous assumption (`ID_public`) was based on the fact that we wouldn't call these functions in generated tests. But I don't think we can completely exclude finding calls to private methods in generated tests, e.g. when mocking private methods (if we do partial mocking in the future), so we need to know consider that the method can be private. Anyway it's too early to make assumptions on how we'll be calling these methods, so we need to stay accurate.
Should I make the comment clearer?